lang/java/avro/src/test/java/org/apache/avro/io/LegacyBinaryEncoder.java

```diff
@@ -178,6 +178,11 @@ public void writeIndex(int unionIndex) throws IOException {
     encodeLong(unionIndex, out);
   }
 
+  @Override
+  public void writeExtends(int extendsIndex) throws IOException {
+    encodeLong(extendsIndex, out);
```

Comment:
Should the extends clause be supported by the legacy encoder at all?

Or should it throw?

Reply:
I don't know ... i can throw an exception ....